test/e2e/framework/util.go

```diff
@@ -5252,3 +5252,37 @@ func WaitForNodeHasTaintOrNot(c clientset.Interface, nodeName string, taint *v1.
 	}
 	return nil
 }
+
+// GetFileModeRegex returns a file mode related regex which should be matched by the mounttest pods' output.
+// If the given mask is nil, then the regex will contain the default OS file modes, which are 0644 for Linux and 0755 for Windows.
+func GetFileModeRegex(filePath string, mask *int32) string {
+	return getFileAttributeRegex("mode", filePath, mask)
+}
+
+// GetFilePermsRegex returns a file permissions related regex which should be matched by the mounttest pods' output.
+// If the given mask is nil, then the regex will contain the default OS file permissions, which are 0644 for Linux and 0755 for Windows.
+func GetFilePermsRegex(filePath string, mask *int32) string {
+	return getFileAttributeRegex("perms", filePath, mask)
+}
+
+func getFileAttributeRegex(attributeName string, filePath string, mask *int32) string {
+	var (
+		linuxMask   int32
+		windowsMask int32
+	)
+	if mask == nil {
+		linuxMask = int32(0644)
+		windowsMask = int32(0775)
+	} else {
+		linuxMask = *mask
+		windowsMask = *mask
+	}
+
+	// forward slashes have to be escaped for regexp
```

Comment:
is this something special imposed by gomega? AFAIK the golang regexp library doesn't treat `/` special.

Reply:
Hm, there might have been a reason why this was done in the 1st place, but I no longer remember. Seems to work without it, so I'll just remove it.